### ISSUE TITLE
Is the :only option working on Rails 4.1?

### ISSUE THREAD

**Issue by josegrad** [stefanpenner/country_select#91]
Hi, I can't get the :only option working. 
```
<%= country_select("address", "country", only: ["GB", "FR", "DE"]) %>
```
Includes all the countries for the select...

I'm on Rails 4.1.8, Ruby 2.1.5
Using the latest release of the gem, 2.1.0

**Comment by scudco** [comment]
I can't reproduce your issue. I added your code snippet to the test app https://github.com/scudco/country_select_test/commit/eb100457ca2fa8defb6daea84faa4953ffca87d1

Do you have the same issue if you boot up the test app?

**Comment by josegrad** [comment]
With the test app it goes ok...
I'll take a closer look.

**Comment by josegrad** [comment]
Ok, Carmen causes a conflict.

I also had gem 'carmen-rails', '~> 1.0.0', github: 'jim/carmen-rails' in the Gemfile

I think I don't need to use Carmen in this project, it came from a branched product so it is legacy.

Problem found and "fixed".

Btw, during my check I also tried the test app with Rails 4.1.8 (you have 4.1.4), it went ok but I had to add config.i18n.available_locales = [:es] to application.rb since it complained about :es no being a valid locale. Just to mention it.
